packages/stark-ui/src/modules/toast-notification/components/toast-notification.component.spec.ts

```diff
@@ -7,8 +7,8 @@ import { MatIconModule } from "@angular/material/icon";
 import { TranslateModule } from "@ngx-translate/core";
 import { STARK_LOGGING_SERVICE } from "@nationalbankbelgium/stark-core";
 import { MockStarkLoggingService } from "@nationalbankbelgium/stark-core/testing";
-import { StarkMessageType } from "../../../common";
-import { StarkToastNotificationComponent } from "../components";
+import { StarkMessageType } from "../../../common/message";
```

Comment:
What is the benefit of importing from common/message and simply from common?

Reply:
In this specific case, there is no difference because the `common` folder just contains 2 classes which have no imports in them.

I just followed the same rule for all this kind of imports because there is an issue in `packages/stark-core/src/common`. Whenever there is an import from "common", then all the classes inside will be loaded but some include some other imports to other folders which at the end cause the "circular dependencies" problem...